src/components/backend-ai-project-switcher.ts

```diff
@@ -66,55 +72,47 @@ export default class BackendAIProjectSwitcher extends LitElement {
    */
   changeGroup(e) {
     globalThis.backendaiclient.current_group = e.target.value;
-    this.currentGroup = globalThis.backendaiclient.current_group;
+    this.currentProject = globalThis.backendaiclient.current_group;
     globalThis.backendaiutils._writeRecentProjectGroup(globalThis.backendaiclient.current_group);
     const event: CustomEvent = new CustomEvent('backend-ai-group-changed', {'detail': globalThis.backendaiclient.current_group});
     document.dispatchEvent(event);
   }
 
-  _refreshUserGroupSelector(): void {
-    this.currentGroup = globalThis.backendaiutils._readRecentProjectGroup();
-    globalThis.backendaiclient.current_group = this.currentGroup;
-    this.groups = globalThis.backendaiclient.groups;
-    // Detached from template to support live-update after creating new group (will need it)
-    if (this.groupSelectionBox?.hasChildNodes()) {
-      this.groupSelectionBox?.removeChild(this.groupSelectionBox.firstChild as ChildNode);
+  _sortProjects(e) {
+    const isAscending = e.target.on;
+    if (isAscending) {
+      this.projects = [...this.projects.sort()];
+    } else {
+      this.projects = [...this.projects.reverse()];
```

Comment:
```suggestion
      this.projects = [...this.projects.sort().reverse()];
```
If there is no 'sort' here, 'this.projects' should already be sorted. It's better to implement it to avoid external dpendancy.